cosmos/launchpad/construction_metadata.go

```diff
@@ -0,0 +1,35 @@
+package launchpad
+
+import (
+	"context"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+)
+
+func (l Launchpad) ConstructionMetadata(ctx context.Context, r *types.ConstructionMetadataRequest) (*types.ConstructionMetadataResponse, *types.Error) {
+	if r.NetworkIdentifier == nil {
```

Comment:
This is already checked by `rosetta-sdk-go` validators :+1: this handler never called if those validations cannot pass.
```suggestion
```

Reply:
Maybe I'm missing something, but I noticed handler is called without network identifier too. 
If it's being validated this should be resulting in error as network identifier is empty right, am I missing something? https://github.com/tendermint/cosmos-rosetta-gateway/blob/7c2c5bca8960ec22e082da5e1d9b5a15dc97507b/cosmos/launchpad/construction_derive_test.go#L27-L29

Reply:
Oh, this test runs as expected because we directly call the handler in the tests. In a real scenario, the handler is actually called by `rossetta-sdk` itself after it runs validation and other intermediate steps. We basically register handlers here for rosetta, they are not directly exposed as http handlers.

Reply:
Ah, got it. It should be fine without this validation then, thanks

Reply:
Yeah not needed.